swap_meet/vendor.py

```diff
@@ -1,2 +1,118 @@
 class Vendor:
-    pass
\ No newline at end of file
+    def __init__(self, inventory = None):
+        '''
+        Its parameter inventory is an empty list
+        '''
+        if not inventory:
+            inventory = []
+        self.inventory  = inventory
+
+    def add(self, add_item):
+        '''
+        It adds item to the inventory.
+        Argument - add_item(string): representing an item
+        Return: the item that was added 
+        '''
+        self.inventory.append(add_item)
+        return add_item
+
+    def remove(self, remove_item):
+        '''
+        It removes the matching item from the inventory.
+        Argument - remove_item(string): representing an item
+        Return: the item that was removed, if there is no matching item it returns False
+        '''
+        if remove_item in self.inventory:
+            self.inventory.remove(remove_item)
+            return remove_item
+        else:
+            return False
```

Comment:
Notice how the primary logic of this function is indented? We can omit the `else` and use a guard clause instead to bring the primary logic more in 'focus'.

```python
    def remove(self, remove_item):
        '''
        It removes the matching item from the inventory.
        Argument - remove_item(string): representing an item
        Return: the item that was removed, if there is no matching item it returns False
        '''
        if remove_item not in self.inventory:
            return False
            
        self.inventory.remove(remove_item)
        return remove_item
```